.github/workflows/deployments.yml

```diff
@@ -263,6 +263,7 @@ jobs:
     strategy:
       matrix:
         platform: ${{ fromJson(needs.metadata.outputs.multi_platform || needs.metadata.outputs.platforms).ids }}
+        cuda_version: ["11.8", "12.0"]
```

Comment:
Do we need similar changes in validation_config.json? They are currently calling out only 11.8.

Reply:
The configuration in validation.json are only relevant for the wheels and the installer, i.e. not technically related to the docker image. The configurations for the wheels don't specify the cuda version, since it will be strictly tested based on what is in the readme (i.e. wheels are tested for both, *if* the runners actually have GPUs - in reality it is only validated during publishing/QA). The configurations you saw I think are the ones for the C++ installer, where the only system with CUDA 12 is debian. I need to check which images support newer cuda versions, and can add them - though again, this is not technically relevant since the runners that use this configuration do not have gpus...